packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts

```diff
@@ -101,14 +104,19 @@ export function getKubernetesClusterQuery(projectId: string, kubeId: string) {
   return ['project', projectId, 'kube', kubeId];
 }
 
-export const useKubernetesCluster = (projectId: string, kubeId: string) =>
+export const useKubernetesCluster = (
+  projectId: string,
+  kubeId: string,
+  refetchIntervalTime?: number,
```

Comment:
As a minor improvement, to avoid excessive parameters, it could be useful to add options like `refetchIntervalTime` as an `options` object that could be easily extended if needed.